### PR TITLE
Fix the provisioning instructions

### PR DESCRIPTION
**Pull request by spanceac**
Changelog-entry: Fix the provisioning instructions for Revolution Pi
Signed-off-by: Sebastian Panceac <sebastian@balena.io>

### CODE REVIEW

**Comment by spanceac** [comment]
@resin-jenkins please retest this

File: revpi-core-3.coffee [hunk 1]
```diff
@@ -10,7 +15,13 @@ module.exports =
 	state: 'released'
 	isDefault: true
 
-	instructions: commonImg.instructions
+	instructions: [
+	REVPI_DEBUG
+	REVPI_POWER
+	REVPI_WRITE
+	REVPI_POWEROFF
+	instructions.CONNECT_AND_BOOT
```
Comment (floion) [inline revpi-core-3.coffee]: Please indent this one and the 4 above it

File: revpi-core-3.coffee [hunk 2]
```diff
@@ -10,7 +15,13 @@ module.exports =
...
+]
```
Comment (floion) [inline revpi-core-3.coffee]: This also needs indentation

**Comment by floion** [comment]
@resin-jenkins test this please

**Comment by spanceac** [comment]
@resin-jenkins test this please

**Pull request closed without merging**